package.json

```diff
@@ -5,7 +5,6 @@
   "homepage": "https://replicache.dev",
   "repository": "github:rocicorp/replicache-react",
   "main": "out/index.js",
-  "module": "out/index.mjs",
```

Comment:
why?

Reply:
I think the revert command did that or something. Wasn't purposeful.